### ISSUE TITLE
installing lQTGui

### ISSUE THREAD

**Issue by asadaku** [cartr/homebrew-qt4#64]
I'm trying to use this to install open rave, which complains at 99% that it can't find lQtGui.a.

I browsed the install directory of qt@4 under libs and there isn't any qtgui stuff. Is there some way to install that too?

**Comment by cartr** [comment]
Sorry to hear you're running into issues with the formula! QtGui should be
included in the qt@4 formula, although I believe we install it as a
framework rather than a static library. Would you mind copy-pasting the
output of "brew doctor", "brew config", and the failing compile command
into a Gist so I can take a look?

**Comment by asadaku** [comment]
Here is the gist. I admit I have been trying like crazy to get this to work so I have installed qt5, qt4 and whatever multiple times in a vain attempt to solve this. I'm so close!!

https://gist.github.com/asadaku/f95f69235ea16c6d82e6922b8230b2e6

**Comment by cartr** [comment]
Hmm. Could you add the output of soqt-config and coin-config?

Carter

**Comment by asadaku** [comment]
The output of soqt-config and coin-config is empty but if I do

soqt-config  --libs
`-lSoQt -F/usr/local/Cellar/qt@4/4.8.7_5/lib -framework QtOpenGL QtGui QtCore -lCoin -lpthread -lQtGui.o -lQtOpenGL.o`

coin-config  --libs
`-lCoin -lpthread`

I did compile these two using your qt formula

**Comment by cartr** [comment]
Yeah, that "-lQtGui.o" probably shouldn't be there. I'll look into this
further.

**Comment by asadaku** [comment]
I should mention how I installed soqt.

```
>> hg clone https://bitbucket.org/Coin3D/soqt -r cmake
>> mkdir build
>> cd build/
>> cmake ../soqt
>> make
>> make install
```

Maybe I am not using a flag that I should be using when building soqt?
